ci-automation/vendor-testing/aws.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Copyright (c) 2022 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+# Test execution script for the AWS vendor image.
+# This script is supposed to run in the mantle container.
+
+source ci-automation/vendor_test.sh
+
+board="${CIA_ARCH}-usr"
+image_name="ci-${CIA_VERNUM//+/-}"
```

Comment:
```suggestion
image_name="ci-${CIA_VERNUM//+/-}-${CIA_ARCH}"
```
Required because ore will otherwise add it by itself for arm64.